src/test/kotlin/dev/stalla/model/Fixtures.kt

```diff
@@ -45,10 +45,5 @@ internal fun anItunesCategory(
 ) = ItunesCategory.from(category)
 
 internal fun aGoogleplayCategory(
-    category: String = "googleplay category",
-    subcategory: String? = "googleplay category subcategory"
-) = if (subcategory == null) {
-    GoogleplayCategory.Simple(category)
-} else {
-    GoogleplayCategory.Nested(category, GoogleplayCategory.Simple(subcategory))
-}
+    category: String = "News & Politics"
```

Comment:
Arguably, we could get rid of this and directly assign an enum value instead where this is used, now that there's no need to instantiate this anymore